js/test/utils.test.js

```diff
@@ -132,103 +135,63 @@ describe('utils', () => {
 
   describe('#sha256', () => {
     it('returns a sha256 hash', () => {
-      let res;
-      let arraysAreEqual;
-      res = utils.sha256(utils.deserializeHex('0x616263'));
-      arraysAreEqual = utils.typedArraysAreEqual(res, utils.deserializeHex('0xba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad'));
-      assert.isTrue(arraysAreEqual);
-
-      res = utils.sha256(new Uint8Array([]));
-      arraysAreEqual = utils.typedArraysAreEqual(res, utils.deserializeHex('0xe3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'));
-      assert.isTrue(arraysAreEqual);
+      for (let i = 0; i < sha256.length; i += 1) {
+        const res = utils.sha256(sha256[i].input);
+        const arraysAreEqual = utils.typedArraysAreEqual(res, sha256[i].output);
+        assert.isTrue(arraysAreEqual);
+      }
     });
   });
 
   describe('#ripemd160', () => {
     it('returns a ripemd160 hash', () => {
-      let res;
-      let arraysAreEqual;
-
-      res = utils.ripemd160(new Uint8Array([0]));
-      arraysAreEqual = utils.typedArraysAreEqual(res, utils.deserializeHex('0xc81b94933420221a7ac004a90242d8b1d3e5070d'));
-      assert.isTrue(arraysAreEqual);
-
-      res = utils.ripemd160(utils.deserializeHex('2c415d8f16ce68ede2b3642800883055'));
-      arraysAreEqual = utils.typedArraysAreEqual(res, utils.deserializeHex('0xb09d9089acb3855efacd76960db70d166c32c5e6'));
-      assert.isTrue(arraysAreEqual);
+      for (let i = 0; i < ripemd160.length; i += 1) {
+        const res = utils.ripemd160(ripemd160[i].input);
+        const arraysAreEqual = utils.typedArraysAreEqual(res, ripemd160[i].output);
+        assert.isTrue(arraysAreEqual);
+      }
     });
   });
 
   describe('#typedArraysAreEqual', () => {
     it('returns true if Uint8Arrays are equal', () => {
-      const arr1 = new Uint8Array([255, 255, 255]);
-      const arr2 = new Uint8Array([255, 255, 255]);
-      const res = utils.typedArraysAreEqual(arr1, arr2);
-      assert.isTrue(res);
-    });
-    it('returns false if Uint8Arrays are not equal', () => {
-      const arr1 = new Uint8Array([255, 255, 254]);
-      const arr2 = new Uint8Array([255, 255, 255]);
-      const res = utils.typedArraysAreEqual(arr1, arr2);
-      assert.isFalse(res);
+      for (let i = 0; i < typedArraysAreEqual.length; i += 1) {
+        const { arr1, arr2 } = typedArraysAreEqual[i].input;
+        const res = utils.typedArraysAreEqual(arr1, arr2);
+        if (typedArraysAreEqual[i].output) {
+          assert.isTrue(res);
+        } else {
+          assert.isFalse(res);
+        }
```

Comment:
could be done simpler as `assert.strictEquals(res, typedArraysAreEqual[i].output)`